sys-devel/slibtool/slibtool-0.5.17.ebuild

```diff
@@ -0,0 +1,17 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="A skinny libtool implementation, written in C"
+
+HOMEPAGE="http://git.midipix.org/cgit.cgi/slibtool"
+SRC_URI="http://git.midipix.org/cgit.cgi/${PN}/snapshot/${P}.tar.xz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm"
+
+src_configure() {
+	./configure --host=${CHOST} --prefix="${EPREFIX}"/usr || die
```

Comment:
Sorry if somebody else asked the question already but why not using `econf` instead of calling configure?